zeroex/ordervalidator/order_validator_test.go

```diff
@@ -435,6 +439,42 @@ func TestComputeOptimalChunkSizesMultiAssetOrder(t *testing.T) {
 	assert.Equal(t, expectedChunkSizes, chunkSizes)
 }
 
+// emptyGetOrderRelevantStatesCallDataByteLength is all the boilerplate ABI encoding required when calling
+// `getOrderRelevantStates` that does not include the encoded SignedOrder. By subtracting this amount from the
+// calldata length returned from encoding a call to `getOrderRelevantStates` involving a single SignedOrder, we
+// get the number of bytes taken up by the SignedOrder alone.
+// i.e.: len(`"0x7f46448d0000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"`)
+const emptyGetOrderRelevantStatesCallDataByteLength = 268
+
+func TestComputeABIEncodedSignedOrderByteLength(t *testing.T) {
+	signedOrder := scenario.NewSignedTestOrder(t)
+	signedOrder.Order.MakerAssetData = common.Hex2Bytes("123412304102340120350120340123041023401234102341234234523452345234")
+	signedOrder.Order.MakerAssetData = common.Hex2Bytes("132519348523094582039457283452")
+	signedOrder.Order.MakerAssetData = common.Hex2Bytes("")
+	signedOrder.Order.MakerAssetData = common.Hex2Bytes("324857203942034562893723452345246529837")
```

Comment:
Let's add more test cases for this function with longer asset data (more than 2 words). This might be a good place to whip out the `multiAssetData`, but I really just want to have more assurances that this function is right across a wider range of inputs.